src/SCSS/details.scss

```diff
@@ -0,0 +1,80 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap');
+
+.details-image {
+  width: 300px;
+  height: 280px;
+}
+
+.details-container {
+  margin-top: 30px;
+  margin-left: 30px;
+}
+
+.button-container {
+  display: flex;
+  height: 50px;
+  width: 10rem;
+  background-color: #98c00d;
+  border-radius: 25px 25px 25px 25px;
+  justify-content: center;
+  align-items: center;
+  margin-top: 5rem;
+}
+
+.reserve-button {
+  background: none;
+  color: white;
+  border: none;
+  padding: 0;
+  font: inherit;
+  cursor: pointer;
+  outline: inherit;
+}
```

Comment:
Please make sure to allow the user to access the reservation form by also clicking on the green part. Currently, the button only works by clicking precisely on the 'Reserve' word.
![image](https://user-images.githubusercontent.com/42799579/198156176-2cc49c27-a547-47f3-8b4e-28bfa554eb0c.png)